lms/static/scripts/postmessage_json_rpc/server/methods.js

```diff
@@ -9,8 +9,22 @@
 /**
  * Return a Hypothesis client config object for the current LTI request.
  */
-export function requestConfig() {
+export async function requestConfig() {
   const configEl = document.querySelector('.js-config');
   const clientConfigObj = JSON.parse(configEl.textContent).hypothesisClient;
-  return clientConfigObj;
+  return Promise.resolve(clientConfigObj);
+}
+
+/**
+ * Temporary method that blocks for a little while to simulate
+ * waiting for canvas groups to be ready.
+ *
+ * TODO: replace this with a real request to lms/
+ */
+export async function groupsAsync() {
```

Comment:
If all of the methods in this file are going to be async, I'd suggest the `Async` suffix can be avoided. I would suggest only adding that if for some reason there are two methods that differ only in whether they are sync or async (like Node's "fs" module has sync and async versions of many functions and adds a `Sync` suffix to the synchronous ones).